### ISSUE TITLE
selector-delimiter-newline-after complaining about nested selectors

### ISSUE THREAD

**Issue by davidtheclark** [stylelint/stylelint#217]
The code:

```css
@define-mixin link {
  color: var(--link-color-base);
  text-decoration: var(--link-decoration);
  transition: color 0.2s linear;

  &:hover,
  &:focus {
    color: var(--link-color-active);
    text-decoration: var(--link-decoration-active);
  }
}

@define-mixin clearfix {
  &::before,
  &::after {
    content: " " !important;
    display: table !important;
  }

  &::after {
    clear: both !important;
  }
}
```

I'm getting complaints about those nested selector delimiters but shouldn't. Should be an easy fix, just have to look into it later.
